v3.19.0: Null Pointer Exception in ExportActivity.

**Describe the bug**
I found this issue:

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{de.dennisguse.opentracks.debug/de.dennisguse.opentracks.io.file.exporter.ExportActivity}: java.lang.NullPointerException: Missing required view with ID: de.dennisguse.opentracks.debug:id/toolbar
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2913)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.NullPointerException: Missing required view with ID: de.dennisguse.opentracks.debug:id/toolbar
        at de.dennisguse.opentracks.databinding.ToolbarBinding.bind(ToolbarBinding.java:59)
        at de.dennisguse.opentracks.databinding.ExportActivityBinding.bind(ExportActivityBinding.java:348)
        at de.dennisguse.opentracks.databinding.ExportActivityBinding.inflate(ExportActivityBinding.java:178)
        at de.dennisguse.opentracks.databinding.ExportActivityBinding.inflate(ExportActivityBinding.java:168)
        at de.dennisguse.opentracks.io.file.exporter.ExportActivity.onCreate(ExportActivity.java:137)
        at android.app.Activity.performCreate(Activity.java:7136)
        at android.app.Activity.performCreate(Activity.java:7127)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2893)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6669) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

PS/ Looks like we have an issue in the Layout because Activity cannot inflate the Layout. If I comment these lines from export_activity.xml then it works:

```xml
<include
        android:id="@+id/toolbar"
        layout="@layout/toolbar" />
```

But I have no idea how to fix it :(

**To Reproduce**
1. Go to Settings -> Export All.
2. Select a folder.
3. See the error.

**Technical information**
 - Device: any
 - OS: any
 - Version: 3.19.0

--- a/src/main/java/de/dennisguse/opentracks/io/file/exporter/ExportActivity.java
+++ b/src/main/java/de/dennisguse/opentracks/io/file/exporter/ExportActivity.java
@@ -37,6 +37,7 @@ import de.dennisguse.opentracks.content.data.Track;
 import de.dennisguse.opentracks.content.data.TracksColumns;
 import de.dennisguse.opentracks.content.provider.ContentProviderUtils;
 import de.dennisguse.opentracks.databinding.ExportActivityBinding;
+import de.dennisguse.opentracks.databinding.ToolbarBinding;
 import de.dennisguse.opentracks.io.file.ErrorListDialog;
 import de.dennisguse.opentracks.io.file.TrackFileFormat;
 import de.dennisguse.opentracks.util.ExportUtils;
@@ -134,6 +135,7 @@ public class ExportActivity extends FragmentActivity implements ExportServiceRes
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         viewBinding = ExportActivityBinding.inflate(getLayoutInflater());
+        ToolbarBinding mergedBinding = ToolbarBinding.bind(viewBinding.getRoot());
         setContentView(viewBinding.getRoot());
 
         directoryUri = getIntent().getParcelableExtra(EXTRA_DIRECTORY_URI_KEY);
@@ -144,7 +146,7 @@ public class ExportActivity extends FragmentActivity implements ExportServiceRes
         DocumentFile documentFile = DocumentFile.fromTreeUri(this, directoryUri);
         String directoryDisplayName = FileUtils.getPath(documentFile);
 
-        viewBinding.toolbar.toolbar.setTitle(getString(R.string.export_progress_message, directoryDisplayName));
+        mergedBinding.toolbar.setTitle(getString(R.string.export_progress_message, directoryDisplayName));
 
         resultReceiver = new ExportServiceResultReceiver(new Handler(), this);
 

--- a/src/main/res/layout/export_activity.xml
+++ b/src/main/res/layout/export_activity.xml
@@ -7,9 +7,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:orientation="vertical">
 
-    <include
-        android:id="@+id/toolbar"
-        layout="@layout/toolbar" />
+    <include layout="@layout/toolbar" />
 
     <androidx.core.widget.NestedScrollView
         android:layout_weight="1"

